.gitignore

```diff
@@ -0,0 +1,6 @@
+manuscript/manuscript.aux
+manuscript/manuscript.dvi
+manuscript/manuscript.log
+manuscript/manuscript.out
+manuscript/manuscript.pdf
```

Comment:
I agree, and as of last month, github renders pdf's in the browser https://github.com/blog/1974-pdf-viewing